src/tools/vs-code.md

```diff
@@ -128,49 +139,54 @@ running or debugging.
 
 ### Run app without breakpoints
 
- 1. Click **Run > Start Without Debugging** in the
-    main IDE window, or press `Ctrl`+`F5`.
+Go to **Run** > **Start Without Debugging** in the main IDE window.
+
+You can also press `<kbd>Ctrl</kbd>`+`<kbd>F5</kbd>`.
 
 ### Run app with breakpoints
 
- 1. If desired, set breakpoints in your source code.
- 1. Click **Run > Start Debugging** in the main IDE window,
-    or press `F5`. The status bar turns orange to show you are in a debug session.<br>
-    ![Debug console]({{site.url}}/assets/images/docs/tools/vs-code/debug_console.png){:.mw-100.pt-1}
-
-    * The left **Debug Sidebar** shows stack frames and variables.
-    * The bottom **Debug Console** pane shows detailed logging output.
-    * Debugging is based on a default launch configuration.
-      To customize, click the cog at the top of the
-      **Debug Sidebar** to create a `launch.json` file.
-      You can then modify the values.
-      
-      
+1. If desired, set breakpoints in your source code.
+1. Click **Run > Start Debugging** in the main IDE window,
+   or press <kbd>F5</kbd>.
+   The status bar turns orange to show you are in a debug session.<br>
+   ![Debug console]({{site.url}}/assets/images/docs/tools/vs-code/debug_console.png){:.mw-100.pt-1}
+
+   - The left **Debug Sidebar** shows stack frames and variables.
+   - The bottom **Debug Console** pane shows detailed logging output.
+   - Debugging is based on a default launch configuration.
+     To customize, click the cog at the top of the
+     **Debug Sidebar** to create a `launch.json` file.
+     You can then modify the values.
+
 ### Run app in debug, profile, or release mode
 
 Flutter offers many different build modes to run your app in. 
 You can read more about them in [Flutter's build modes][].
 
- 1. Open the `launch.json` file in VS Code.
-    
-    If you do not have a `launch.json` file, go to 
-    the **Run** view in VS Code and click **create a launch.json file**.
- 1. In the `configurations` section, change the `flutterMode` property to 
- the build mode you want to target. 
-     * For example, if you want to run in debug mode, 
-     your `launch.json` might look like this: 
-     ```json
-      "configurations": [
-       {
-         "name": "Flutter",
-         "request": "launch",
-         "type": "dart",
-         "flutterMode": "debug"
-       }
-     ]
-     ```
- 1. Run the app through the **Run** view. 
+1. Open the `launch.json` file in VS Code.
+
+   If you do not have a `launch.json` file, go to
+   the **Run** view in VS Code and click **create a launch.json file**.
```

Comment:
This might need updating. I wasn't sure what **Run** view is. Perhaps "**Run and Debug** tag"?